src/platform/tls_schannel.c

```diff
@@ -1003,7 +1037,12 @@ CxPlatTlsSspiNotifyCallback(
         CompletionCallback(&CredConfig, CompletionContext, SecStatusToQuicStatus(Status), NULL);
         CxPlatTlsSecConfigDelete(SecConfig); // *MUST* be last call to prevent crash in platform cleanup.
     } else {
-        CompletionCallback(&CredConfig, CompletionContext, QUIC_STATUS_SUCCESS, SecConfig);
+        QUIC_STATUS Status = QUIC_STATUS_SUCCESS;
```

Comment:
`##[warning]src\platform\tls_schannel.c(1040,28): Warning C4456: declaration of 'Status' hides previous local declaration`